packages/workflows/src/definition/price-list/index.ts

```diff
@@ -0,0 +1,5 @@
+export * from "./create-price-lists"
+export * as UpdatePriceLists from "./update-price-lists"
```

Comment:
not sure why we have an alias for this UpdatedPriceLists workflow `UpdatePriceLists.updatePriceLists`
they should all be either exported on the main package, or bundled in a single place, like `import { updatePriceLists } from `@medusajs/workflows/dist/core`